metadata/OrgSetup/standardValueSets/LPIIncomeFrequency.standardValueSet

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
How come in recurrence you have bi-weekly, but here you don't?

Reply:
Updated! I removed bi-weekly and made IncomeFrequency and Expense RecurrenceInterval values same, as it was recommended to have only 2-3 picklist values as this is sample data